<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find::Internal::FindWidget</class>
 <widget class="QWidget" name="Find::Internal::FindWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="findLabel">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::FilterLineEdit" name="findEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="findButtonsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="findPreviousButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="findNextButton">
        <property name="font">
         <font/>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="close">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="replaceLabel">
       <property name="text">
        <string>Replace with:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Utils::FilterLineEdit" name="replaceEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="replaceButtonsWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="replaceButtonsLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="replaceButton">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Replace</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="replaceNextButton">
          <property name="font">
           <font/>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Replace &amp;&amp; Find</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="replaceAllButton">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Replace All</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="advancedButton">
         <property name="text">
          <string>Advanced...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::FilterLineEdit</class>
   <extends>Utils::FancyLineEdit</extends>
   <header location="global">utils/filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>findEdit</tabstop>
  <tabstop>replaceEdit</tabstop>
  <tabstop>close</tabstop>
  <tabstop>replaceAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
